source/utilities.tex

```diff
@@ -2476,13 +2479,11 @@
 
 \pnum
 \expects
-Each element in \tcode{*this} is swappable with\iref{swappable.requirements}
-the corresponding element in \tcode{rhs}.
+For all $i$, \tcode{get<$i$>(*this)} is swappable with\iref{swappable.requirements} \tcode{get<$i$>(rhs)}.
 
 \pnum
 \effects
-Calls \tcode{swap} for each element in \tcode{*this} and its
-corresponding element in \tcode{rhs}.
+For each $i$, calls \tcode{swap} for \tcode{get<$i$>(*this)} with \tcode{get<$i$>(rhs)}.
```

Comment:
I didn't notice this earlier, but "calls `swap` for `x` with `y`" is not how we'd usually say it. I would expect "swaps x with y" but when talking about a call, call "`swap` for `x` *and* `y`" seems better to me. This is what LWG approved though.

Reply:
Let's make this an issue!